rest-server/src/components/auth/authRouter.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import validate from 'express-validation';
+import validation from '../../lib/request-validation'
+import Auth from './authController';
```

Comment:
Let's destructure whenever possible, and since we'll probably be importing more from these controllers, let's have them on multiple lines

```javascript
import {
  signUp
} from './authController'
```